triton-vm/src/table/hash_table.rs

```diff
@@ -321,17 +61,21 @@ impl Evaluable for ExtHashTable {
         evaluation_point: &[XFieldElement],
     ) -> Vec<XFieldElement> {
         let round_number = evaluation_point[ROUNDNUMBER as usize];
+        let state10 = evaluation_point[STATE10 as usize];
+        let state11 = evaluation_point[STATE11 as usize];
         let state12 = evaluation_point[STATE12 as usize];
         let state13 = evaluation_point[STATE13 as usize];
         let state14 = evaluation_point[STATE14 as usize];
         let state15 = evaluation_point[STATE15 as usize];
 
-        let round_number_is_not_1_or = (0..=8)
+        let round_number_is_not_1_or = (0..=9)
             .filter(|&r| r != 1)
             .map(|r| round_number - r.into())
             .fold(1.into(), XFieldElement::mul);
 
         let mut evaluated_consistency_constraints = vec![
+            round_number_is_not_1_or * (state10 - XFieldElement::one()), // <-- domain separation bit
```

Comment:
Is the domain separation bit present even when hashing input of known, fixed length?

Reply:
Clarified by f490f554764d852896f2c1066d9576515ff7b96e.